### PR TITLE
update raft

### PR DESCRIPTION
**Pull request by Presburger**

### CODE REVIEW

File: src/index/ivf_raft/ivf_raft.cuh [hunk 1]
```diff
@@ -437,45 +441,10 @@ class RaftIvfIndexNode : public IndexNode {
...
-            raft::serialize_mdspan(*res_, os, gpu_index_->centers());
-            if (gpu_index_->center_norms()) {
-                bool has_norms = true;
-                serialize_scalar(*res_, os, has_norms);
-                serialize_mdspan(*res_, os, *gpu_index_->center_norms());
-            } else {
-                bool has_norms = false;
-                serialize_scalar(*res_, os, has_norms);
-            }
+            raft::neighbors::ivf_flat::serialize<float, std::int64_t>(*res_, os, *gpu_index_);
```
Comment (liliu-z) [inline src/index/ivf_raft/ivf_raft.cuh:444]: Wow, newly supported in RAFT?

**Review by liliu-z — APPROVED** [review]
/lgtm

File: src/index/ivf_raft/ivf_raft.cuh (hunk 1 above)
Comment (Presburger) [reply src/index/ivf_raft/ivf_raft.cuh:444]: yes

**Pull request merged**